src/views/PIGEAN/Phenotype/Template.vue

```diff
@@ -204,7 +228,7 @@
         <page-footer :disease-group="$parent.diseaseGroup"></page-footer>
     </div>
 </template>
-<style scoped>
+<style>
```

Comment:
you probably want <style module> here if you want to style content from v-html. Removing scoped will make them global, and might/might not create conflicts with other styles.